app/jobs/notify_email_job.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require "notifications/client"
+
+class NotifyEmailJob < ApplicationJob
+  class NotConfiguredError < RuntimeError; end
+
+  NOTIFY_TEMPLATE_ID = "7cb31359-e913-4590-a458-3d0cefd0d283"
```

Comment:
Should we put this in a configuration variable or something? We might want to use a different notify account per local authority, for example. (Though I'm not sure if it matters so much for emails.)